style.css

```diff
@@ -535,3 +545,314 @@ input {
   background: white;
   color: #5e6c84;
 }
+
+@media only screen and (min-width: 992px) {
+  body {
+    background: #e5e5e5;
+    max-width: 100%;
+  }
```

Comment:
- [ ] Hi well done on making the changes from the previous reviewer 👍👍. I can see you've adjusted some of your CSS to ensure that the design is flexible on the desktop version. however, your mobile version is not looking correct. You should ensure that both your desktop and mobile version are working with each other. try to avoid using a fixed-width when you want something to be responsive
- [ ] On the headline section ensure that the width spans the entire mobile version you don't want to give it a fixed width
![image](https://user-images.githubusercontent.com/18730106/144680041-68fb9d34-2d49-45da-baaf-5755d9e1cf6a.png)
- [ ] [OPTIONAL] Please for the mobile version center the cards so they are in the center of the page. The Figma design expectation is that the content is centered.
![image](https://user-images.githubusercontent.com/18730106/144680266-a1476e6d-f3a7-4131-be4a-9d27fe33e99a.png)
- [ ] Make sure that this "About Myself" sections width also spans the entire mobile version width.  
![image](https://user-images.githubusercontent.com/18730106/144680444-34d5bedb-3de1-4350-a8cd-34e7d977daab.png)
- [ ] This last "Contact" section as well should span the entire mobile view before it gets to the desktop version
![image](https://user-images.githubusercontent.com/18730106/144680584-f7c916a3-575e-4d6b-8de5-b4420bcc72bc.png)